server/api/absence-record/absence-record.controller.js

```diff
@@ -17,23 +17,6 @@ exports.index = function(req, res) {
 };
 
 /**
- * Get list of absence records by school
- * restriction: 'teacher'
- */
-exports.bySchool = function(req, res) {
-  School.find({name: req.query.school}, function(err, schools) {
-    if (err) { return handleError(res, err); }
-    var schoolID = schools[0]._id;
-    console.log(schoolID);
-    AbsenceRecord.find({school: schoolID}, function(err, records) {
-      var result = records[0];
-      console.log(result);
-      return res.status(200).json(result);
-    });
-  });
-};
-
-/**
  * Get a single absence record
```

Comment:
It's not being used. Also looking up the school shouldn't be needed for getting the absence record. We can add this back when we need this route in the future.